### DISCUSSION TITLE
Prior art

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/vocab-idl#41] [General]
Let's collect a list of all the code generation tools we can find that use JSON Schema.

Found a new one (to me) today https://github.com/microsoft/jschema

**Comment by jonaslagoni** [comment]
- https://quicktype.io/
- https://github.com/asyncapi/modelina/
- https://github.com/OpenAPITools/openapi-generator
- https://www.npmjs.com/package/json-schema-to-typescript

**Comment by notEthan** [comment]
https://github.com/notEthan/jsi is ... kind of a code generator. It generates classes and mixins and methods, but in the runtime, not in source code. I think that is consistent with the goals of this project, though a different approach.

**Comment by mwadams** [comment]
I'll add [Corvus.JsonSchema](https://github.com/corvus-dotnet/Corvus.JsonSchema) - a dotnet code generator that doesn't use inheritance, but does support polymorphic conversions.
